### ISSUE TITLE
[NTR]: Z chromosome 

### ISSUE THREAD

**Issue by mikewhitelab** [Tree-of-Sex/tos-ontology#20]
### In which category is your new term request?

- [ ] new_definition
- [X] existing_definition

### Preferred term label

Z chromosome

### Textual definition

The sex chromosome present in both sexes of species in which the female is the heterogametic sex. Two copies of the Z chromosome are present in each somatic cell of males and one copy is present in females. 
go:0000807

### Suggested parent term

chromosome
sex chromosome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by enemeshazi** [comment]
Approved. I would remove 'go:0000807' from the definition and add the reference under 'additional context'.

**Comment by Lucija27** [comment]
I agree with the definition "The sex chromosome present in both sexes of species in which the female is the heterogametic sex." Should maybe the second part, i.e., "Two copies of the Z chromosome are present in each somatic cell of males and one copy is present in females. go:0000807", be added into the description or a comment?

**Comment by laiamaringual** [comment]
I agree with the definition and both suggestions

**Comment by CEMcDonoughGoldstein** [comment]
At the most recent ontology meeting we talked about ways to make the descriptions of these chromosomes sex-neutral. i.e., can we define these chromosomes without a reference to sex and allow the association of chromosomes configurations and sex to come from the observations.

With that I was trying also to think of a parent term that could be used to describe both Y and W or X and Z

Mank et al. 2012 [10.1007/s10577-011-9251-2](https://doi.org/10.1007/s10577-011-9251-2) used major sex chromosome (X or Z) and minor sex chromosome (Y and W)

So the definitions could be something like:

minor sex chromosome - a chromosome that when present is associated with the heterogametic sex
major sex chromosome - a chromosome that is present in two copies in most of the somatic cells of the homogametic sex and one copy in most of the somatic cells heterogametic sex
another version major sex chromosome - a chromosome that is found in the somatic cells of both sexes in systems with chromosomal sex determination with two copies in the homogametic sex and one copy in the heterogametic sex

Z chromosome - a type of major sex chromosome that is associated with female heterogametic sex systems

I think the major/minor has some unwanted associations about more and less important that aren't ideal so would be interested in other suggestions or thoughts about options.

**Comment by CEMcDonoughGoldstein** [comment]
Instead of minor sex chromosome for Y and W could have the term be sex-limited or sex-restricted or sex-specific sex chromosome

and for major sex chromosome X and Z could be shared sex chromosome, common sex chromosome, non-restricted or non-limited sex chromosome

**Comment by KamilSJaron** [comment]
Was the idea avoid X, Z, Y and W altogether? I would say the abstracted terms are super useful. "Sex limited chromosome" is my favourite for Ys and Ws. Less opinionated about the other one, "Shared sex chromosome" does not sound bad, although I never heard anyone using it. I would probably call it "Homogametic sex chromosome", but that might be misleading too.
